index.js

```diff
@@ -0,0 +1,30 @@
+const formElement = document.querySelector('.editForm');
+const  nameInput = document.querySelector('#nameInput');
+const  jobInput = document.querySelector('#jobInput');
+const  formButton = document.querySelector('.editForm__button');
+const  formCloseIcon = document.querySelector('.editForm__close-icon');
+const  infoTitle = document.querySelector('.profile-info__title');
+const  infoSubtitle = document.querySelector('.profile-info__subtitle');
+const  editButton = document.querySelector('.profile-info__edit-button');
+
+
+editButton.addEventListener('click', (evt) => {
```

Comment:
положи 'click' в константу EVENTS

```suggestion
editButton.addEventListener(EVENTS.CLICK, (evt) => {
```